Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,12 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo su mastodon`.
+
+This command will create the user as needed:
+
+    sudo adduser --disabled-password --gecos "" mastodon
```

Comment:
Perhaps add:
- `--system` to get a system user (different UID range);
- `--group` to get a `mastodon` group as primary GID for that user;
- a `--homedir` parameter, rather than making the user's home under `/home`;
- use `--disabled-login` rather than `--disabled-password` (with the latter, one can still login with SSH keys and such).